samples/Ring/hello.ring

```diff
@@ -0,0 +1 @@
+See "Hello, World"
```

Comment:
@pchaigno Thanks, The file is updated to include a sample from "real" applications under permissive license.